src/components/spinner/index.tsx

```diff
@@ -14,7 +14,7 @@ export class SmoothlySpinner {
 		return (
 			<Host
 				style={{
-					"--color": `var(--spinner-color)`,
+					"--text-color": `var(--spinner-color)`,
```

Comment:
is this intended?

Reply:
No.

Reverting the file to its original state.